src/Data/API/FileFetcherImpl.test.ts

```diff
@@ -0,0 +1,12 @@
+import { BaseApiHelper } from "./BaseApiHelper";
+import { FileFetcherImpl } from "./FileFetcherImpl";
+
+test("GIVEN FileFetcher WHEN executing get THEN the result is correctly returned", async () => {
+  const fileFetcher = new FileFetcherImpl(new BaseApiHelper(), "env");
+  fetchMock.mockResponse(`{"content": "abcdefgh"}`);
```

Comment:
I do not really see a reason for using the fetchMock api here.
Th only reason for using that would be to test the BaseApiHelper.
Here I would just use our DeferredApiHelper.
Then you can also assert on the request being done, increasing the test coverage.

Reply:
I don't think it improves the coverage that much, but it is more consistent with the other test cases. I applied the changes